app/bundles/CoreBundle/Assets/js/1.core.js

```diff
@@ -895,3 +895,13 @@ var Mautic = {
         }
     }
 };
+
+function listQuickFilter(element) {
```

Comment:
The code for this function seems fine, but the place and formatting does not fit within the Mautic JS structure:

- It shouldn't be in core.js because it's not part of the core setup. This file is (ideally) for setting up events and default Ajax behaviour. The Quickfilters is an extra functionality for a specific type of component (part of search functionality). Maybe the `6.forms.js` file is a better place for it? Or a new file called `search.js`? See if there's any JS functionality related to the search/filtering and in what file that lives.
- The function declaration doesn't follow the standard used in core.js nor in the other JS files. It's not the only function that does it wrong (eg. `initializeCodeBlocks` is also not following standards) but we should follow standard as much as possible. Normally, the functions are declared within the `Mautic` namespace, so you should do the same unless there's a specific reason not to do so.

Eg. if it's in core.js, you'll see the functions are nested within the declaration on Line 130. 
IF it's in another JS file, you'll see functions are declared by prefixing `Mautic.`, eg. `Mautic.renameFormElements = function(` in `6.forms.js`

Reply:
Got it!
I’ll address the formatting 

The file seems correct because it’s used for more than forms (it adds for users too) and can be extended to other lists bc of the generic/flexible way it was implemented

But I’m going to have a look at other files to ensure

Reply:
Moved to 1a.content.js